core/vm/interpreter.go

```diff
@@ -254,8 +254,7 @@ func (in *EVMInterpreter) Run(contract *Contract, input []byte, readOnly bool) (
 		// consume the gas and return an error if not enough gas is available.
 		// cost is explicitly set so that the capture state defer method can get the proper cost
 		if operation.dynamicGas != nil {
-			var dynamicCost uint64
-			dynamicCost, err = operation.dynamicGas(in.evm, contract, stack, mem, memorySize)
+			dynamicCost, err := operation.dynamicGas(in.evm, contract, stack, mem, memorySize)
```

Comment:
This will assign two new variables instead of only one (reassigning the return value `err`). Is this just a pretty-fication change?